substra/sdk/client.py

```diff
@@ -721,6 +721,54 @@ def download_objective(self, key: str, destination_folder: str) -> None:
             os.path.join(destination_folder, 'metrics.py'),
         )
 
+    @logit
+    def download_model(self, key: str, folder) -> None:
+        """Download model to destination file."""
+        self._backend.download_model(key, os.path.join(folder, f'model_{key}'))
+
+    @logit
+    def download_traintuple_model(self, tuple_key: str, folder) -> None:
```

Comment:
Could you add some pointers on how to use the model ? Like `The model has been saved with the 'save_model' function of the traintuple algorithm. To load and use the model, please refer to the 'load_model' and 'predict' functions of the algorithm` (feel free to adapt)

Reply:
fixed https://github.com/SubstraFoundation/substra/pull/261/commits/cd04734d6c64cd9d970d1ea90c93fb8b79ab1b70